projects/angular/src/layout/_card.clarity.scss

```
@@ -170,27 +187,78 @@
     }
   }
 
-  .card-header,
-  .card-block,
+  .card-block {
+    @include mixins.css-var(
+      padding,
+      clr-card-block-padding,
+      card-variables.$card-children-padding-vertical card-variables.$card-children-padding-horizontal,
+      variables.$clr-use-custom-properties
+    );
+  }
+
   .card-footer {
-    padding: card-variables.$card-children-padding-vertical card-variables.$card-children-padding-horizontal;
+    display: flex;
+    flex-direction: row;
+    @include mixins.css-var(
+      gap,
+      clr-card-footer-gap,
+      variables.$clr_baselineRem_0_5,
+      variables.$clr-use-custom-properties
+    );
```

Wouldn't this change affect ng theme? Isn't it better to use `button-variables.$clr-btn-horizontal-margin` for fallback?
```suggestion
    @include mixins.css-var(
      gap,
      clr-card-footer-gap,
      button-variables.$clr-btn-horizontal-margin,
      variables.$clr-use-custom-properties
    );
```

---

I copied the value which `clr-btn-horizontal-margin` is using so it basically the same and we don't have to import the button-variables for this specific value. The footer is not specifically made for buttons only so It's confusing to use the buttons margin.